libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -29,10 +34,14 @@ import type {
   ChatReadRequest,
   TypedCommsResponse
 } from './clientTypes'
+import WebSocket from 'isomorphic-ws'
+import EventEmitter from 'events'
+import type TypedEmitter from 'typed-emitter'
 
 export class ChatsApi extends BaseAPI {
   private chatSecrets: Record<string, Uint8Array> = {}
   private readonly walletApi: WalletAPI
+  public eventEmitter: TypedEmitter<ChatEvents>
```

Comment:
maybe instead of making this public, I should have `ChatsApi` implement the emitter and proxy the `on` method?

Reply:
implementing `on` probably better since it means clients can't abuse this event emitter and emit their own events, etc